Libraries/Blob/URL.js

```diff
@@ -43,13 +42,143 @@ if (BlobModule && typeof BlobModule.BLOB_URI_SCHEME === 'string') {
  *
  * ```xml
  * <resources>
- *   <string name="blob_provider_authority">your.app.package.blobs</string>
+ *   <string name='blob_provider_authority'>your.app.package.blobs</string>
  * </resources>
  * ```
  */
-class URL {
-  constructor() {
-    throw new Error('Creating URL objects is not supported yet.');
+
+// Small subset from whatwg-url: https://github.com/jsdom/whatwg-url/tree/master/lib
+// The reference code bloat comes from Unicode issues with URLs, so those won't work here.
+export class URLSearchParams {
+  searchParams = [];
```

Comment:
This doesn't seem to be a valid property according to the spec. Maybe prefix it with `_` mark as private?